.github/workflows/deepx-ci.yml

```diff
@@ -0,0 +1,28 @@
+name: DeepX CI
+
+run-name: Build Docker container and perform unit testing
+
+on: [push, workflow_dispatch]
+
+jobs:
+  build-library:
+    runs-on: ubuntu-20.04
+    name: Build and test
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Login to Github Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: deepx-inc
+          password: ${{ github.token }}
+
+      - name: Build and commit the Docker container
+        run: |
+          docker build --build-arg BASE_IMAGE=ghcr.io/deepx-inc/base_images --build-arg TAG=foxy -t deepx-gz-rendering .
+      
+      - name: Run tests
+        run: |
+          docker run deepx-gz-rendering:latest bash -c 'cd /root/gazebo/gz-rendering/build; cmake ..; make install/; ctest;'
```

Comment:
We've already built the library. We should remove the `cmake ..` and `make install` here, or remove the building in the dockerfile.

Reply:
I tried without using `cmake ..` and `make install` here, and either it didn't work, or I didn't use it correctly. So I removed the building in the dockerfile. The last part is now simply `RUN mkdir -p gz-rendering/build`.